nimx/image.nim

```diff
@@ -83,7 +83,7 @@ when not defined js:
             of 2: GL_LUMINANCE_ALPHA
             of 3: GL_RGB
             of 4: GL_RGBA
-            else: GLenum(0)
+            else: GL_FALSE
```

Comment:
This fixes the following error:
`/Users/matthew/.nimble/pkgs/nimx-0.1/nimx/image.nim(86, 25) Error: type mismatch: got (GLenum) but expected 'int literal(6406)'`